script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_1.txt

```diff
@@ -6,20 +6,20 @@ The sound of creaking vehicles.
 Artificial lights passing by at regular intervals.
 As the tremors of the seat overlap with the second hand of the clock, I think about the passage of time and space.
 <Page3>
-It's early morning, 6:33 am.
+It's early morning, 6:33 AM.
 Half an hour has passed since I got on this train.
 I can't go back anymore, all this time and distance is now keeping me apart from the family that took care of me for all these years.
 How I accept this may be a measure of the depth of my compassion as a human being.
 <Page4>
-Do you think of them as being 'still that close'?
+Do I think of them as being 'still that close'?
```

Comment:
I wonder if the changes in this file should just be scrapped? a lot of them are conflicting and we bailed out early during group PR.